x/gov/keeper/querier_test.go

```diff
@@ -316,13 +316,24 @@ func TestPaginatedVotesQuery(t *testing.T) {
 	app.GovKeeper.SetProposal(ctx, proposal)
 
 	votes := make([]types.Vote, 20)
-	rand := rand.New(rand.NewSource(time.Now().UnixNano()))
-	addr := make(sdk.AccAddress, 20)
+	random := rand.New(rand.NewSource(time.Now().UnixNano()))
+	addrMap := make(map[string]struct{})
+	genAddr := func() string {
+		addr := make(sdk.AccAddress, 20)
+		for {
+			random.Read(addr)
+			if _, ok := addrMap[addr.String()]; ok {
```

Comment:
We need to invert this and make it simpler to read and even less.

for {
random.Read(addr)
addrStr := addr.String()
if _, ok := addrMap[addrStr]; !ok {
     addrMap[addrStr] = struct{}{}
     return addr
}
}

Reply:
Done